workflow/pred_temperature_ridge.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<p>An emphasis about the procedure we take for preprocessing is added here. I agree that it is necessary to echo that we are using daily data and perform preprocessing operation at this stage, not only because it works in this way, but also for standardization. Thanks for your reminder.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Some minor textual changes</p><p> </p><p>In general, there are many "flavors" of preprocessing, like when to perform this operation, and in which order do we want to preprocess the data. To improve the transparency and reproducibility of our work, we think it is necessary to standardize these steps. To stick to the best practices, we suggest to preprocess your data in the following way.</p>